.gitignore

```diff
@@ -89,12 +89,10 @@ egeria-lineage-repositories/**
 *-graph-repository/**
 
 # Ignore archives
-CloudInformationModel.json
-CloudInformationModelGUIDMap.json
-DataStoreConnectorTypes.json
-DataStoreConnectorsGUIDMap.json
-OpenMetadataTypes.json
+*.json
```

Comment:
I wonder if this exclude is rather too general -- will we not have .json files in future?

If the files are GENERATED they should be under the target tree - those files are already entirely excluded from git.

See below for unit test too - we should generate them into a target directory

This helps keep excludes under control, and also makes it easier for packaging and assembly

Reply:
Another reason for this is that other build lifecycle aspects like 'clean' will work properly without extra work, as well as making it easier to run other utilities across the source - for docs etc.

Unfortunately in both the UI and the subject area tests this wasn't done which caused a fair bit of heartache to fix up some consequential issues.

Reply:
@planetf1 I agree normally. I have changed the logic so the json file is generated from the model name - so we do not know it advance. It is the existing logic for the archive writer to write the output here - I have not changed that. As far as my change -there is a json file which is the archive which is picked up from here; this is the same as a the CIM model. There is a guid file is generated (also existing logic) - but you would not necessarily want it deleted as part of a maven clean - as it is the file that ensures subsequent builds use the same guids.    I suggest we raise a separate issue to review the input and output files locations as I see this as a separate issue; we can amend the .gitignore at that stage if required. Is that ok with you? 
I will look at the junit as that should write to target.